test/Macros/PointerBounds/CountedBy/MutableSpan.swift

```diff
@@ -0,0 +1,16 @@
+// REQUIRES: swift_swift_parser
+// REQUIRES: pointer_bounds
+
+// RUN: %target-swift-frontend %s -swift-version 5 -module-name main -disable-availability-checking -typecheck -plugin-path %swift-plugin-dir -dump-macro-expansions 2>&1 | %FileCheck --match-full-lines %s
+
+@PointerBounds(.countedBy(pointer: 1, count: "len"), .nonescaping(pointer: 1))
+func myFunc(_ ptr: UnsafeMutablePointer<CInt>, _ len: CInt) {
+}
+
+// CHECK:      @_alwaysEmitIntoClient
+// CHECK-NEXT: func myFunc(_ ptr: MutableSpan<CInt>) {
```

Comment:
MutableSpan isn't actually defined anywhere. Not sure why the Windows tests are crashing here, but at this point it isn't expected to really work.